app/src/main/res/layout/item_currency.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="@style/CardView.Light"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="8dp"
+    android:layout_marginTop="8dp"
+    android:layout_marginEnd="8dp"
+    android:layout_marginBottom="8dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/card_background"
+        android:id="@+id/currency_background">
+
+        <ImageView
+            android:id="@+id/currencyImage"
+            android:layout_width="128dp"
+            android:layout_height="0dp"
+            android:layout_marginStart="16dp"
```

Comment:
what do you think about using dimens.xml resources to set dimens in layout files instead of hardcoding them? :D

Reply:
Sounds good, thanks!